src/sass/components/_card.scss

```diff
@@ -161,18 +161,31 @@
     }
 
     &__candidate-read-more-link {
-      left: 0;
       margin-left: 10px;
       top: 0;
-      font-weight: lighter;
+      font-family: $body-font-stack;
+      font-size: 14px;
+      font-weight: 100 !important;
```

Comment:
@DaleMcGrew be mindful of this extremely light weight. I would actually recommend against using such a light weight for small text like this. There is the even possibility you are not even seeing the actual weight as it doesn't render the 100 weight for me on desktop – I only noticed it on my iPhone.